### PR TITLE
SIMD i8x16.neg / i16x8.neg / i32x4.neg / i64x2.neg implementation.

### PR DESCRIPTION
**Pull request by lizhengxing**

### CODE REVIEW

**Comment by lizhengxing** [comment]
@binji 
SIMD neg instructions implementation. PTAL, thanks!

@arunetm FYI!

File: src/interp.cc [hunk 1]
```diff
@@ -1047,6 +1070,12 @@ ValueTypeRep<R> IntEqz(ValueTypeRep<T> v_rep) {
   return ToRep(v_rep == 0);
 }
 
+// i{32,64}.neg
```
Comment (binji) [inline src/interp.cc]: nit: this isn't a real instruction, so it's better to just remove this comment.

File: src/opcode.def [hunk 2]
```diff
@@ -254,6 +254,10 @@ WABT_OPCODE(V128, V128, V128, ___, 0, 0xfd, 0x1f,  I64X2Sub, "i64x2.sub")
 WABT_OPCODE(V128, V128, V128, ___, 0, 0xfd, 0x20,  I8X16Mul, "i8x16.mul")
 WABT_OPCODE(V128, V128, V128, ___, 0, 0xfd, 0x21,  I16X8Mul, "i16x8.mul")
 WABT_OPCODE(V128, V128, V128, ___, 0, 0xfd, 0x22,  I32X4Mul, "i32x4.mul")
+WABT_OPCODE(V128, V128, ___, ___, 0, 0xfd, 0x23,  I8X16Neg, "i8x16.neg")
```
Comment (binji) [inline src/opcode.def]: nit: align columns please!

File: src/interp.cc (hunk 1 above)
Comment (lizhengxing) [reply src/interp.cc]: Done.

File: src/opcode.def (hunk 2 above)
Comment (lizhengxing) [reply src/opcode.def]: Done.

**Comment by lizhengxing** [comment]
@binji 
updated the PR for windows platform. PTAL, thanks!

@arunetm FYI!

**Pull request merged**